src/helpers/pullOutValues.js

```diff
@@ -0,0 +1,13 @@
+const pullOutValues = (data) => {
```

Comment:
`pullOutValues` seems like a name that says little about intent. What about `object2string` or `objectSerialization`?

Reply:
object2string ins't a good one because the function doesn't tranform an object to string. However, objectSerialization is a good fit.